### DISCUSSION TITLE
Is there something like ` overflow-y: scroll; ` for overflow of widgets and paragraphs, separately for each tab!

### DISCUSSION THREAD

**Discussion by ArnavK-09** [ratatui/ratatui#1110] [Q&A]
Sorry, I'm new to ratatui and have 0 knowledge 
I tried using 3rd party lib, but tab content isn't rendering well

**Comment by joshka** [comment]
There's a crate I wrote [tui-scrollview](https://crates.io/crates/tui-scrollview), which might help. In it you'll still need to work out the actual size of your content before rendering.

**Reply by ArnavK-09** [reply]
Well I tried this crate, but somehow I was unable to create a proper scrollable view for each tab, it was just showing Scrollbar but tab content isn't visible! Is there any example for it? Independent Scrollbar for each of the tab.

Regards

**Reply by joshka** [reply]
Take a look at https://github.com/joshka/tui-scrollview/blob/main/examples/tabs.rs for a PoC of this. There's probably some simpler approaches you could take, but this works.
